### PR TITLE
Add quotes to prevent the shell from looking for a local folder/file

### PR DESCRIPTION
**Pull request by nhoizey**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 A command-line tool helping the choice of optimal responsive image widths to put in your `srcset` attribute(s).
 
-Install as a global package: `npm install -g cleverage/responsive-image-widths#master`
+Install as a global package: `npm install -g "cleverage/responsive-image-widths#master"`
```
Comment (ryuran) [inline README.md]:
```suggestion
Install as a global package: `npm i -g github:cleverage/responsive-image-widths#master`
```
maybe specify it’s github in this case

Comment (nhoizey) [reply README.md]: I still get this error if I add `github:` but remove the quotes:

`zsh: no matches found: github:cleverage/responsive-image-widths#master`

It really looks like zsh try to find a `github:cleverage/` folder…

Comment (nhoizey) [reply README.md]: 6.4.1

Comment (nhoizey) [reply README.md]: It looks like it’s related to `zsh` or `oh-my-zsh`:
https://github.com/robbyrussell/oh-my-zsh/issues/31#issuecomment-359728582

Let’s keep it without quotes.

**Pull request merged**